edb/ir/typeutils.py

```diff
@@ -120,6 +120,16 @@ def is_bytes(typeref: irast.TypeRef) -> bool:
     return typeref.real_base_type.id == s_obj.get_known_type_id('std::bytes')
 
 
+def needs_text_serialization(typeref: irast.TypeRef) -> bool:
+    return (
+        typeref.real_base_type.id == s_obj.get_known_type_id('fts::language'))
+
+
+def needs_semitext_serialization(typeref: irast.TypeRef) -> bool:
```

Comment:
Why not fold these into a single function?

Reply:
Hm, we could. I think of them as being conceptually distinct, but all the existing cases where we use `needs_text_serialization` could be `needs_semitext_serialization` since the cases can be distinguished by context.